src/main/webapp/app/overview/result-history/result-history.component.html

```diff
@@ -1,20 +1,22 @@
 <div class="result-history-wrapper ms-3">
-    <div class="result-history-element previous-history-element" *ngIf="showPreviousDivider">
+    <div class="result-history-element" [class.previous-history-element]="showPreviousDivider">
         <div class="result-divider"></div>
     </div>
     <div class="result-history-element" *ngFor="let result of displayedResults; let i = index">
-        <div class="result-score" [ngbTooltip]="result.completionDate | artemisDate" placement="top" container="body">
+        <div class="result-score">
             <div class="result-score-icon" [ngClass]="resultClass(result)">
                 <fa-icon [icon]="resultIcon(result)"></fa-icon>
             </div>
             <jhi-result
                 class="result-score-info text-center"
+                [style.gap]="'4px'"
```

Comment:
Should we keep styling information in such a format? Do we do that somewhere else? I rather suggest working with classes here. A child component should be more contained.